tekton/ci.yaml

```diff
@@ -52,65 +52,61 @@ spec:
       workspaces:
         - name: source
           workspace: ws
-    - name: unit-tests-ppc64le
```

Comment:
I removed these unit tests because we can't run unit tests on a different platform/arch.